yarn.lock

```diff
@@ -992,6 +999,156 @@
     through2 "^2.0.0"
     xdg-basedir "^3.0.0"
 
+"@material-ui/core@^3.9.3":
```

Comment:
Overkill having reactstrap and material for this project. We should stick to reactstrap since its the library that used the most in our project

Reply:
yarn remove this package